cfme/tests/containers/test_ad_hoc_metrics.py

```diff
@@ -16,6 +17,13 @@
                          scope='function')
 ]
 
+TEST_ITEMS = [ContainersProviderCollection, NodeCollection]
+
+
+def get_test_object(appliance, provider, test_item):
+    return (
+        provider if test_item is ContainersProviderCollection else test_item(appliance).all().pop())
```

Comment:
And instead here:
```return(provider if test_item == 'containers_providers' else getattr(appliance.collections, test_item).all().pop())```

What do you think? We can get rid of those imports.

Reply:
@jawatts Personally, I prefer the long line, but I will change this if you want...

Reply:
@shalomnaim1 @jawatts the important distinction here is the instantiation of the collection class. Collection classes should not be instantiated directly through `__new__`, but with a call to `appliance.collections` - in this case through `getattr()` since the collection name (the attribute of `appliance.collections`) is in a variable.